### PR TITLE
config(pd): update pd master required context

### PR DESCRIPTION
**Pull request by purelind**
Update pd master required context. releate to https://github.com/tikv/pd/pull/7991.

### CODE REVIEW

File: prow/config/config.yaml [hunk 1]
```diff
@@ -1316,7 +1308,23 @@ tide:
...
-              master: &pd-branch-tide-context-options
+              master:
+                required-contexts:
+                  - "statics"
+                  - "chunks (1, Unit Test)"
+                  - "chunks (2, Tools Test)"
+                  - "chunks (3, Client Integration Test)"
+                  - "chunks (4, TSO Integration Test)"
+                  - "chunks (5, MicroService Integration Test)"
+                  - "tso-function-test"
```
Comment (wuhuizuo) [inline prow/config/config.yaml:1319]: We can write it like `tidb` to keep simple.

File: prow/config/config.yaml [hunk 2]
```diff
@@ -28,19 +28,11 @@ branch-protection:
...
-                  - "chunks (9)"
-                  - "chunks (10)"
-                  - "chunks (11)"
-                  - "chunks (12)"
-                  - "chunks (13)"
+                  - "chunks (1, Unit Test)"
+                  - "chunks (2, Tools Test)"
+                  - "chunks (3, Client Integration Test)"
+                  - "chunks (4, TSO Integration Test)"
+                  - "chunks (5, MicroService Integration Test)"
```
Comment (HuSharp) [inline prow/config/config.yaml:35]:
```suggestion
                  - "chunks (1, Unit Test(1))"
                  - "chunks (2, Unit Test(2))"
                  - "chunks (3, Tools Test)"
                  - "chunks (4, Client Integration Test)"
                  - "chunks (5, TSO Integration Test)"
                  - "chunks (6, MicroService Integration Test)"
```

Comment (HuSharp) [reply prow/config/config.yaml:35]: https://github.com/tikv/pd/pull/7991 PTAL, thx!

File: prow/config/config.yaml [hunk 3]
```diff
@@ -1316,7 +1309,23 @@ tide:
...
             branches:
-              master: &pd-branch-tide-context-options
+              master:
+                required-contexts:
+                  - "statics"
+                  - "chunks (1, Unit Test)"
+                  - "chunks (2, Tools Test)"
+                  - "chunks (3, Client Integration Test)"
+                  - "chunks (4, TSO Integration Test)"
+                  - "chunks (5, MicroService Integration Test)"
```
Comment (HuSharp) [inline prow/config/config.yaml:1334]:
```suggestion
                  - "chunks (1, Unit Test(1))"
                  - "chunks (2, Unit Test(2))"
                  - "chunks (3, Tools Test)"
                  - "chunks (4, Client Integration Test)"
                  - "chunks (5, TSO Integration Test)"
                  - "chunks (6, MicroService Integration Test)"
```

**Review by HuSharp — APPROVED** [review]

**Review by wuhuizuo — APPROVED** [review]

**Pull request merged**